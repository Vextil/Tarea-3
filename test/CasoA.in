CrearLista 1 a b c d e f g h i FIN
CrearBalanceado 1 1
ImprimirBinario 1
CrearBinario o 1 1 k 2 a 3 n 4 r 5 y 6 u 7 FIN
ImprimirBinario 1
Linealizar 1 1
ImprimirLista 1
FiltroBinario 1 2 < 2
ImprimirBinario 2
ConstruirSet 1 1
CopiarSet 1 2
CrearLista 2 a k n o r FIN
RemoverDeSet 2 2
PerteneceASet 2 1
UnionInterDif 1 2 3 4 5 6
PerteneceASet 6 1
AgregarProcesos notepad++ 5 word 50 thunderbird 20 chrome 999999 FIN
ListarProcesos
MuyConsumidores 9999
IngresarSolicitudes chrome 1 notepad++ 2 thunderbird 1 word 3 FIN
EnAlgunoSolicitudes 1 3 1
CrearLista 1 thunderbird word chrome notepad++ FIN
PerteneceASet 1 1
IngresarSolicitudes chrome 3 FIN
EnAmbosSolicitudes 1 3 1
PerteneceASet 1 1
Fin
