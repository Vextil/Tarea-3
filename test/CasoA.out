1>OK
2>OK
3>
   (0,i)
  (0,h)
   (0,g)
    (0,f)
 (0,e)
   (0,d)
  (0,c)
   (0,b)
    (0,a)
4>OK
5>
   (6,y)
    (7,u)
  (5,r)
 (1,o)
   (4,n)
  (2,k)
   (3,a)
6>OK
7>Ordenada. a k n o r u y 
8>OK
9>
   (6,y)
    (7,u)
  (5,r)
 (4,n)
  (3,a)
10>OK
11>OK
12>OK
13>OK
14>NO NO NO NO NO SI SI 
15>OK
16>NO NO NO NO NO NO NO 
17>Hay 5 procesos.
18>boot chrome notepad++ thunderbird word 
19>chrome 
20>OK
21>OK
22>OK
23>SI SI SI NO 
24>OK
25>OK
26>NO NO SI NO 
27>Fin
